client/src/pages/CreateConversation.js

```diff
@@ -75,15 +67,22 @@ class CreateConversation extends Component {
         errorMessage: "Please record audio"
       });
     } else {
-      // success!
-      console.log("Conversation created! (umm soon)");
+      const userId = window.localStorage.getItem("id");
+      const data = new FormData();
+      data.append('audio', this.state.blobObject.blob);
+      data.set('title', this.state.title);
 
-      // TODO
-      // Save audio to user and amazon S3
-      //direct user to profile page
-      this.props.history.push("/profile", { id: this.props.location.state.id });
+      await axios
+        .put(`/api/users/conversations/${userId}`, data)
```

Comment:
why is the user id in this API request?

and if it was, it would be users/:userId/conversations

Also this should be a POST request
But you shouldn't need it

Reply:
I am storing the ID references of conversation documents inside an array of a user document. I use the ID to retrieve a user.